src/index.test.js

```diff
@@ -653,7 +653,7 @@ describe('<SuperTreeview />', () => {
             expandToggleBtn = component
                 .find('.super-treeview')
                 .find('TransitionGroup')
-                .childAt(0)
+                .childAt(1)
```

Comment:
Will add a test that checks that the first transition group (node) does not have an expand button, due to introduced default behaviour